js/piwik.js

```diff
@@ -4266,7 +4266,7 @@ if (typeof window.Piwik !== 'object') {
             function logEvent(category, action, name, value, customData, callback)
             {
                 // Category and Action are required parameters
-                if (trim(String(category)).length === 0 || trim(String(action)).length === 0) {
+                if (typeof category !== 'undefined' || typeof action !== 'undefined' || trim(String(category)).length === 0 || trim(String(action)).length === 0) {
```

Comment:
Shouldn't that be a `=== 'undefined'` ?

Reply:
You are right. It seems like I forgot to flip them.